src/api/rest/server/server.go

```diff
@@ -421,7 +428,7 @@ func RunServer(port string) {
 	g.GET("/:nsId/testGetAssociatedObjectCount/:resourceType/:resourceId", rest_mcir.RestTestGetAssociatedObjectCount)
 
 	selfEndpoint := os.Getenv("SELF_ENDPOINT")
-	apidashboard := " http://" + selfEndpoint + "/tumblebug/swagger/"
+	apidashboard := " http://" + selfEndpoint + "/tumblebug/api/index.html"
```

Comment:
```suggestion
	apidashboard := " http://" + selfEndpoint + "/tumblebug/api"
```
/tumblebug/api 만으로도 redirect가 된다면, 간단하게 표현해주는 것이 좋을 것 같습니다. :)